editoast/openapi.yaml

```diff
@@ -895,6 +895,13 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/ElectricalProfile"
+        404:
+          description: Electrical profile not found
+          content:
+            application/json:
+              schema:
+                type: object
+                $ref: '#/components/schemas/ApiError'
```

Comment:
We didn't specify this for other endpoints since it's not parsed by rtk. I don't think it's needed since all endpoint errors will follow the same schema.